charts/nutanix-csi-snapshot/Chart.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: v2
+type: application
+version: 1.0.0
+kubeVersion: ">= 1.17.0-0"
+appVersion: "1.0.0"
+name: nutanix-csi-snapshot
+description: Snapshot components required for CSI snapshotting and not specific to any CSI driver
+home: https://github.com/nutanix/helm
+maintainers:
+- name: nutanix-cloud-native-bot
+  email: cloudnative@nutanix.com
+icon: https://avatars2.githubusercontent.com/u/6165865?s=200&v=4
+keywords:
+  - Nutanix
+  - Storage
+  - Snapshot
+  - SnapshotClass
+  - RedHat
+  - CentOS
+  - Ubuntu
+annotations:
+  artifacthub.io/displayName: "Nutanix CSI Snapshot"
+  artifacthub.io/containsSecurityUpdates: "true"
+  artifacthub.io/changes: |
+    - Update Snapshot Controller to 3.0.3
```

Comment:
3.0.3 or 4.2.1

Reply:
As we manage both now I will remove the change info

Reply:
fixed in commit 7d1d1f9